src/administration/first_steps.md

```diff
@@ -31,6 +31,7 @@ To be included in the list of Lemmy instances on [join-lemmy.org](https://join-l
 - [x] Federate with at least one instance from the list
 - [x] Have a site description and icon
 - [x] Don't have closed sign-ups
+- [x] Make sure you require registration applications. This helps keep bots and trolls from using your server to spam the fediverse.
```

Comment:
Should be merged with the previous point (dont have closed signups). Or rather remove that one.

Reply:
Done.